stablehlo/dialect/VhloTypes.td

```diff
@@ -228,6 +228,71 @@ def VHLO_UniformQuantizedV1 : VHLO_TypeDef<"UniformQuantizedV1", "quant_v1", "0.
   }];
   let assemblyFormat = "`<` $storageType `` `:` `` $expressedType `,` $scale `` `:` `` $zeroPoint `,` $storageTypeMin `` `:` `` $storageTypeMax `,` $flags `>`";
 }
+def VHLO_QuantizationScalesV1 : ArrayRefParameter<"::llvm::APFloat", "array of double scales"> {
```

Comment:
Can we just use ArrayRefParameter without the indirection?

Reply:
The [mlir::FieldParser](https://github.com/llvm/llvm-project/blob/6b42625b1f983f6aafb9f4fe2953970c73963603/mlir/include/mlir/IR/DialectImplementation.h#L62) does not have any specialization for `mlir::APFloat` and hence we needed to prove the parser support for that. IMO, [here](https://github.com/openxla/stablehlo/blob/b0edcdd6810551bf6aaec7d5aa00451cfe207582/stablehlo/dialect/VhloTypes.td#L196) where we cannot reply on the `FeildParser`'s existing specialization for parsing. 

For the ZeroPoints, the indirection can be removed as  `FieldParser<int>::parse` is supported.

Reply:
Hmm. We have parsing/printing defined for `VHLO_APFloatV1`. I wonder if there's some way to leverage that. `ArrayRefParameter<VHLO_APFloatV1>` or `ArrayRefParameter<"vhlo::APFloatV1Type">`

Reply:
Yes, I tried to re-use the APFloatV1, but it did not work. Unlike the other VHLO_Typedef (like VHLO_BooleanV1), there is no APFloatV1Type. Similarly, no type defined with VHLO_APFloatV1.

Reply:
This doesn't have implicaitons on bytecode I suppose, since it will get written as an APFloat so lgtm.